package.json

```diff
@@ -25,14 +25,16 @@
   ],
   "devDependencies": {
     "@types/chalk": "^0.4.31",
+    "ava": "^0.19.1",
+    "nyc": "^11.0.2",
     "tslint": "^5.4.3",
     "typescript": "2.2.0"
   },
   "scripts": {
     "build": "tsc && mv dist/bin/denali.js dist/bin/denali && chmod +x dist/bin/denali && echo 'Build successful'",
     "prepublish": "in-publish && npm run build || not-in-publish",
-    "lint": "tslint --project tsconfig.json -c tslint.json --type-check | sed \"s|$PWD/||\"",
-    "test": "npm run lint"
+    "lint": "! tslint --project tsconfig.json -c tslint.json --type-check | sed \"s|$PWD/||\" | grep ERROR",
```

Comment:
It negates the result of the line...e.g. if grep finds nothing, the script exits with 0 because the content is empty. If grep does find something then it exits with 1, halting execution.